TAB2MXL/src/main/java/View/StringParserUtilityBass.java

```diff
@@ -2,33 +2,51 @@
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Hashtable;
+import java.util.Map;
+import java.util.Stack;
 
 import TAB2MXL.BassNoteUtility;
 import TAB2MXL.Measure;
 import TAB2MXL.Note;
 import TAB2MXL.NoteUtility;
 
-
-//Can handle a 4 string bass
-
-public final class StringParserUtilityBass {
+public class StringParserUtilityBass {
```

Comment:
Is it possible to make this extend stringparserutility? just so we dont recycle any code